io-package.json

```
@@ -302,7 +313,7 @@
                 "role": "level",
                 "read": true,
                 "write": true,
-                "max": 100,
+                "max": 105,
```

Change this back to 100 because you are extending the Object in main.js and setting the max to 105 at that point.
```suggestion
                "max": 100,
```

---

Done